tests/core/apt/task.yaml

```diff
@@ -1,16 +1,23 @@
 summary: Ensure that the apt output on UC16 is correct
 
-# TODO:UC20: test that apt-get doesn't exist on UC18 / UC20 in this tests and
-#            re-enable
-systems: [ubuntu-core-16-*]
-
 execute: |
-    expected="Ubuntu Core does not use apt-get, see 'snap --help'!"
-    if apt-get update > output.txt; then
-        echo "apt should exit 1 but did not"
-        exit 1
-    fi
-    if [ "$(cat output.txt)" != "$expected" ]; then
-        echo "Unexpected apt output: $(cat output.txt)"
-        exit 1
+    # shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB/systems.sh"
+
+    if is_core16_system; then
+        # UC16 has apt-get output a message
+        expected="Ubuntu Core does not use apt-get, see 'snap --help'!"
+        if apt-get update > output.txt; then
+            echo "apt should exit 1 but did not"
+            exit 1
+        fi
+        if [ "$(cat output.txt)" != "$expected" ]; then
+            echo "Unexpected apt output: $(cat output.txt)"
+            exit 1
+        fi
+    else 
+        # UC18+ does not have apt-get at all
+        if command -v apt-get; then
+            echo "apt-get command exists on Ubuntu Core, but should not!"
```

Comment:
In this case we also should exit 1, right?

Reply:
Yes you're right I missed this one